tsconfig.json

```diff
@@ -1,11 +1,13 @@
 {
   "extends": "@tsconfig/node12/tsconfig.json",
   "compilerOptions": {
+    "resolveJsonModule": true,
     "noImplicitAny": true,
     "outDir": "./dist"
   },
   "include": [
-    "src/**/*"
+    "src/**/*",
+    "package.json"
```

Comment:
I'm not sure this is necessary, I believe that the "package.json" will always be included by default.

Reply:
solved